run-play.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+ansible-playbook -u vgs-vault-dev -b -K -i '10.10.31.237,10.10.20.61,10.10.32.171,10.10.2.183,10.10.22.213,10.10.30.5,10.10.32.173,10.10.20.89,10.10.21.215,10.10.22.151,10.10.20.225,10.10.20.17,10.10.20.169,10.10.31.217,10.10.21.196,10.10.32.160,10.10.30.126,10.10.30.163,10.10.30.63,10.10.32.78' playbook.yml
```

Comment:
Hardcoded?

Reply:
sorry we should fix that!

Reply:
i'm doing it like this:
```
$ AWS_DEFAULT_REGION=us-east-1 env $(awsudo vault-prod | grep AWS | xargs) ansible-playbook -i inventories/vg-pci -l vault plays/aws-inspector.yml
```
or to run it on bastion only:
```
env $(awsudo vgs-dev | grep AWS | xargs) ansible-playbook -i inventories/vg-pci -l bastion -e ansible_port=2022 plays/aws-inspector.yml
```